src/app/connect/account-abstraction/infrastructure/page.mdx

```diff
@@ -49,7 +49,16 @@ With a thirdweb API key, you get access to bundler and paymaster infrastructure
 | B3                   | ❌      | ✅      |
 | Plume                | ❌      | ✅      |
 | Camp Network         | ❌      | ✅      |
-| Vanar                | ❌      | ✅      |
+| Vanar                | ✅      | ✅      |
+| DFK                  | ❌      | ✅      |
+| Form                 | ❌      | ✅      |
+| Cyber                | ✅      | ✅      |
+| Treasure Ruby        | ✅      | ✅      |
+| Redstone             | ✅      | ✅      |
+| Klaytn Cypress       | ✅      | ✅      |
+| OpBNB                | ✅      | ✅      |
+| Nautilus             | ✅      | ✅      | 
```

Comment:
For Nautilus, Fuse, OpBnB, Treasure Ruby, -- what are their testnets?

Reply:
Removed testnets from each, I do not see where we have enabled AA for their testnets, only mainnet. Sorry about that